extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/AuthRuntimeConfig.java

```diff
@@ -24,6 +25,16 @@ public class AuthRuntimeConfig {
     @ConfigItem(name = "policy")
     public Map<String, PolicyConfig> rolePolicy;
 
+    /**
+     * Properties file containing the client certificate common name (CN) to role mappings.
+     * Use it only if the mTLS authentication mechanism is enabled with either
+     * `quarkus.http.ssl.client-auth=required` or `quarkus.http.ssl.client-auth=request`.
+     * <p/>
+     * Properties file is expected to have the `CN:role1,role,...,roleN` format.
```

Comment:
I know the `.properties` format also supports `:` as a key/value separator, but we use `=` everywhere AFAIK, so we should do here as well I think?

```suggestion
     * Properties file is expected to have the `CN=role1,role,...,roleN` format.
```

Reply:
@Ladicek Sure, this is also used in the tests and suggested here, https://github.com/quarkusio/quarkus/issues/23683#issuecomment-1818385631, so not sure where I found `:`, and good to know it is actually expected to work with Properties :-)